modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/DefaultPageHeader.scss

```
@@ -3,6 +3,14 @@
 .lfr-ddm__default-page-header {
 	margin-bottom: 2rem;
 
+	&-back-button {
+		padding-left: 0;
+
```

You can add a `position: absolute` to avoid the title to move whenever the back button is displayed?
```suggestion
position: absolute;

```

---

Hi @miltonmc , I've put this to test it but I didn't get it to put in the mockup's position. The padding and margin I was putting was not working, so I think it's ok to go on without it.